NoJones/Controller/DashboardViewController.swift

```diff
@@ -89,11 +90,19 @@ class DashboardViewController: InitialScreenViewController {
         
         noAddictionMessage.frame.size.height = self.addictions.isEmpty ? 220 : 0
```

Comment:
Quanto for usar magic number como nesses casos: ``` 220 ``` é bom tu explicar isso em um Enum da vida, ou uma struct,  exemplo

```
enum Constants: CGFloat {
    case dashBoardTableViewHeaderHeight = 2200
}
```

caso de uso: ```  tableView.heightForHeader = Constants.dashBoardTableViewHeaderHeight.rawValue ```

Vantagem: Tu não cria code smell de magic number no teu código, que depois vai dar muito trabalho de manter